pxtrunner/runner.ts

```diff
@@ -328,7 +328,8 @@ namespace pxt.runner {
                     const docsUrl = pxt.webConfig.docsUrl || '/--docs';
                     let verPrefix = mp[2] || '';
                     let url = mp[3] == "doc" ? `${mp[4]}` : `${docsUrl}?md=${mp[4]}`;
-                    window.open(verPrefix + url, "_blank");
+                    let fullUrl = (url.indexOf('/') == 0) ? verPrefix + url : verPrefix + "/" + url;
```

Comment:
```
verPrefix.replace(/\/$/, '') + '/' + ...
```

Reply:
Issue is in some cases the url starts with "/" and in other cases it doesn't. 
verPrefix is authored by us, and so we control that. I don't add a "/" at the end of verPrefix.